<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterFoodPlaceWindow</class>
 <widget class="QMainWindow" name="RegisterFoodPlaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="subir_foto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>401</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ubicacion">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ubicación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crear_puesto">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar Puesto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cerrar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesión</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre_puesto">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre del Puesto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comentarios">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
